_data/icons/razornetwork.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url": "ipfs://QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK",
+      "width": 1024,
+      "height": 1024,
```

Comment:
size seems to be wrong
```
identify QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK         03:34:28
QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK PNG 900x900 900x900+0+0 8-bit sRGB 47186B 0.000u 0:00.000
```

Reply:
This is resolved, I have updtedupdated the dimensions to `900x900`